controllers/machinehealthcheck_controller_test.go

```diff
@@ -522,43 +518,22 @@ func TestClusterToMachineHealthCheck(t *testing.T) {
 	// BEGIN: Set up test environment
 	g := NewWithT(t)
 
-	testEnv = &envtest.Environment{
-		CRDs: []runtime.Object{
-			external.TestGenericBootstrapCRD,
-			external.TestGenericBootstrapTemplateCRD,
-			external.TestGenericInfrastructureCRD,
-			external.TestGenericInfrastructureTemplateCRD,
-		},
-		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases")},
-	}
-
-	var err error
-	cfg, err := testEnv.Start()
+	testEnv, err := helpers.NewTestEnvironment()
```

Comment:
Looking in `NewTestEnvironment`, it seems we're trying to make a generic catch-all functions to create test environments, although the `CRDDirectoryPaths` will be different depending which package we're testing

Reply:
I think we can just include all the CRDs all the time